From aef80b38fd14b3a8d2c97bb97e5a9652c1fcbd71 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:38 -0400
Subject: [usb] core/hcd: moved asterix to variable

Message-id: <1410887463-20674-27-git-send-email-dzickus@redhat.com>
Patchwork-id: 95131
O-Subject: [RHEL7 PATCH 026/591] drivers: usb: core: hcd: moved asterix to variable
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 842c19600728dc2561f06553e442031fc68c1882
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Sat Oct 5 18:02:06 2013 +0200

    drivers: usb: core: hcd: moved asterix to variable

    instead of type

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hcd.c b/drivers/usb/core/hcd.c
index 9795a21..a063e48 100644
--- a/drivers/usb/core/hcd.c
+++ b/drivers/usb/core/hcd.c
@@ -2824,7 +2824,7 @@ void usb_remove_hcd(struct usb_hcd *hcd)
 EXPORT_SYMBOL_GPL(usb_remove_hcd);
 
 void
-usb_hcd_platform_shutdown(struct platform_device* dev)
+usb_hcd_platform_shutdown(struct platform_device *dev)
 {
  struct usb_hcd *hcd = platform_get_drvdata(dev);
 
-- 
1.7.1